files/common/var/lib/delphix-platform/dynamic-debug

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+#
+# Enable specific pr_debug() statements in the kernel to aid in debugging
+# potential issues. The debug statements can only be enabled on loaded modules,
+# so we manually load some modules that aren't loaded by default.
+#
+modprobe target_core_mod
+modprobe iscsi_target_mod
+cat <<-EOF >/sys/kernel/debug/dynamic_debug/control
+	# iSCSI debug info related to LUN RESETs and iSCSI reconnects.
+	file target_core_tmr.c +p
+	func iscsit_release_commands_from_conn +p
+EOF
+
+echo "pr_debug statements enabled:"
+grep '=p' /sys/kernel/debug/dynamic_debug/control
+
+#
+# We do not want to fail the service if some of the dynamic debug fails
```

Comment:
Why not? If we're OK with this silently failing, do we really need to add it at all?

IMO, it's preferable to fail completely, than allow silent degradation of services; in this case, the degradation might not be customer visible, but it definitely would be visible to us if we had to root cause an issue which relied on these messages (and they were missing).

Reply:
So the reason for this is that different kernels may have different messages present, functions may be renamed, there may be differences between platforms, etc. While those messages are useful for debugging potential issues, I'd not want them to add more friction to the process, especially given that the Delphix Engine may be running an older version of the kernel and a newer version of the userland during deferred upgrades.

Reply:
Hm.. I see. OK, I'm fine as is.